client/tests/e2e/navFooter.spec.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-undef */
+
+/// <reference types='Cypress' />
+
+describe('Use the Navigation', () => {
+  it('Visits the Home Page', () => {
+    cy.visit('/home');
+    cy.get('h2').should('contain', 'Home');
+  });
+  it('navigates to the My Shelves Page', () => {
+    cy.visit('/home');
+    cy.get('[href="/myshelves"]').click();
+    cy.url().should('include', '/myshelves');
```

Comment:
Have a look at https://github.com/testing-library/cypress-testing-library. It adds some great new methods to find elements e.g. by aria role: `cy.findByRole('link')` or `cy.findByRole('heading', {level: 2})`. It could help you writing even better tests.